tests/crm/conftest.py

```diff
@@ -142,6 +144,51 @@ def set_polling_interval(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     time.sleep(wait_time)
 
 
+def get_intf_list(duthost, tbinfo, enum_frontend_asic_index):
```

Comment:
The "duthost" used here could be different dut than the random duthost used in other functions of the test(For example: https://github.com/sonic-net/sonic-mgmt/blob/1bbf95f726ca9f4c055ac56dca824b8048696268/tests/crm/conftest.py#L74) on multi-dut testbed such as dualtor, need to align them.